devproject/css/style.css

```diff
@@ -0,0 +1,162 @@
+*{
+	margin: 0 auto 0 auto;
+	text-align:left;
+}
+
+body{
+	Margin:0;
+	padding:0;
+	text-align:left;
+	font-size:13px;
+	font-family:arial,helvatica,sens-serif;
+	color:black;

```

Comment:
Using color names is easy but they are so few. Learn how color hex codes work. The editor should help you do that.